shop-info-service/src/functions/getProductsList/index.ts

```diff
@@ -0,0 +1,14 @@
+import { handlerPath } from '@libs/handler-resolver';
+
+export const getProductsList = {
+  handler: `${handlerPath(__dirname)}/handler.getProductsList`,
+  events: [
+    {
+      http: {
+        method: 'get',
+        path: 'get-products-list',
```

Comment:
/products

Reply:
From the requirements: "3 - The getProductsById AND **getProductsList** lambda functions return a correct response code (POINT2)"
So I think "get-products-list" is sutable here